lib/inputstreamhelper/kodiutils.py

```diff
@@ -106,7 +119,9 @@ def yesno_dialog(heading='', message='', nolabel=None, yeslabel=None, autoclose=
     from xbmcgui import Dialog
     if not heading:
         heading = ADDON.getAddonInfo('name')
-    return Dialog().yesno(heading=heading, line1=message, nolabel=nolabel, yeslabel=yeslabel, autoclose=autoclose)
+    if kodi_version_major() < 19:
+        return Dialog().yesno(heading=heading, line1=message, nolabel=nolabel, yeslabel=yeslabel, autoclose=autoclose)
```

Comment:
We should probably move this to using kodiutils as well.

Reply:
I don't understand, what do you mean exactly?

Reply:
The kodiutils library is an abstraction layer to all Kodi-related calls, and here we aren't using it.

Reply:
I'm confused, this code is in **kodiutils.py**

Reply:
@dagwieers You probably didn't see this is inside kodiutils?

Reply:
Ok, I was confused. I thought we moved everything to kodiutils in the past, so I was surprised to see this :smile: Move along, nothing to see here...